ReactCommon/cxxreact/JSCExecutor.cpp

```diff
@@ -113,7 +113,7 @@ static JSValueRef nativeInjectHMRUpdate(
     const JSValueRef arguments[],
     JSValueRef* exception) {
   String execJSString = Value(ctx, arguments[0]).toString();
-  String jsURL = Value(ctx, arguments[1]).toString();
+  String jsURL = argumentCount > 1 ? Value(ctx, arguments[1]).toString() : String();
```

Comment:
would you mind just adding a comment for our future selves, something like

`temporary workaround for 0.57 and Metro 48`

so that if I encounter any issue in future releases/cherry-picks I can quickly remember what this was about?

Reply:
Done :-)